tests/end2end/features/spell.feature

```diff
@@ -0,0 +1,38 @@
+# vim: ft=cucumber fileencoding=utf-8 sts=4 sw=4 et:
+
+Feature: Setting spell checking for QtWebEngine
```

Comment:
Those should be skipped if no dict is available (like when running tests locally)

Reply:
Sorry for newer answering this - but you got it right from a quick look!